### ISSUE TITLE
Uncaught SyntaxError: Unexpected number

### ISSUE THREAD

**Issue by marty80** [jakesankey/git-history#14]
[Enter steps to reproduce below:]
1. I opened a file called model.cfc (also directory.cfc), then
2. ran Show File History

**Atom Version**: 0.188.0
**System**: Microsoft Windows 7 Enterprise 
**Thrown From**: [git-history](https://github.com/jakesankey/git-history) package, v2.3.0
### Stack Trace

Uncaught SyntaxError: Unexpected number

```
At file:///C:/Users/220983F/AppData/Local/atom/app-0.188.0/resources/app/static/index.html#%7B%22locationsToOpen%22%3A%5B%7B%22pathToOpen%22%3Anull%7D%5D%2C%22bootstrapScript%22%3A%22C%3A%5C%5CUsers%5C%5C220983F%5C%5CAppData%5C%5CLocal%5C%5Catom%5C%5Capp-0.188.0%5C%5Cresources%5C%5Capp%5C%5Csrc%5C%5Cwindow-bootstrap.js%22%2C%22resourcePath%22%3A%22C%3A%5C%5CUsers%5C%5C220983F%5C%5CAppData%5C%5CLocal%5C%5Catom%5C%5Capp-0.188.0%5C%5Cresources%5C%5Capp%22%2C%22devMode%22%3Afalse%2C%22safeMode%22%3Afalse%2C%22appVersion%22%3A%220.188.0%22%2C%22shellLoadTime%22%3A408%2C%22initialPaths%22%3A%5B%22C%3A%5C%5CUsers%5C%5C220983F%5C%5Ccoldfusion%5C%5Cwsiteadm%5C%5Cweb%22%5D%7D:1

SyntaxError: Unexpected number
  at Object.parse (native)
  at stdout (C:\Users\220983F\.atom\packages\git-history\lib\git-history-view.coffee:40:13)
  at Socket.<anonymous> (C:\Users\220983F\AppData\Local\atom\app-0.188.0\resources\app\src\buffered-process.js:148:13)
  at Socket.emit (events.js:138:20)
  at Pipe.close (net.js:477:12)

```
### Commands

```
     -2:01.9 tree-view:expand-directory (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
 12x -2:01.3 core:move-down (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
  4x -1:59.9 core:move-up (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -1:58.1 tree-view:expand-directory (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
  4x -1:56.9 core:move-down (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -1:53.5 tree-view:open-selected-entry (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -1:45.6 git-history:show-file-history (atom-text-editor.editor)
     -1:09.8 command-palette:toggle (atom-text-editor.editor)
     -1:04.0 git-history:show-file-history (atom-text-editor.editor)
     -0:51.5 tree-view:collapse-directory (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:51.0 core:move-up (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:50.5 tree-view:open-selected-entry (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:47.0 git-plus:menu (atom-text-editor.editor)
     -0:43.6 git-history:show-file-history (atom-text-editor.editor)
     -0:36.0 settings-view:open (atom-text-editor.editor)
     -0:08.3 git-history:show-file-history (atom-text-editor.editor)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  },
  "git-history": {}
}
```
### Installed Packages

``` coffee
# User
git-go, v1.0.0
git-history, v2.3.0
git-log, v0.3.0
git-plus, v4.4.10
language-cfml, v0.5.1

# Dev
No dev packages
```

**Comment by ChowSinWon** [comment]
hi @marty80 
do you experience those problems only with all *.cfc files or just with those two files mentioned?
or with all files?

i could not reproduce the problem but i _can_ think of a case where you might run into a problem when having double quotes in your git user name. this would mean you never get any file histories.
i will have a fix for that soon.

**Comment by jakesankey** [comment]
#15 fixed #14  .. Thanks @ChowSinWon
